src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random(System.currentTimeMillis());

```

Comment:
I think this is not needed
```suggestion
    private final Random random = new Random();

```